frontend/conf/routes

```diff
@@ -91,6 +91,7 @@ GET            /patrons                               controllers.Info.patron
 GET            /supporter                             controllers.Info.supporter
 GET            /us/supporter                          controllers.Info.supporterUSA
 GET            /eu/supporter                          controllers.Info.supporterEurope
+GET            /intl/supporter                        controllers.Info.supporterInternational
```

Comment:
which other link are you comparing it to? `theguardian.com/int/supporter` is a 404, theguardian.co.uk redirects to `http://www.theguardian.com/uk `... where is the `/int/supporter`?